Package.swift

```diff
@@ -34,8 +29,22 @@ let package = Package(
     ],
     products: [
         .library(name: "RevenueCat",
-                 targets: ["RevenueCat"])
+                 targets: ["RevenueCat"]),
+        .library(name: "ReceiptParser",
+                 type: .static,
+                 targets: ["ReceiptParser"])
     ],
     dependencies: dependencies,
-    targets: resolveTargets()
+    targets: [
+        .target(name: "RevenueCat",
+                dependencies: [.byName(name: "ReceiptParser")],
```

Comment:
In my experience, this will complicate things quite a bit. 
It's a total headache to manage imports from a local package, we had something like this back in the objc days with PurchasesCoreSwift and it broke hybrids in all sorts of ways. 

I think we should just have these two be independent packages, which just happen to have the same files. 
If you use RC, you don't need the ReceiptParser package. If you use the ReceiptParser package, and you want to move to the full RevenueCat, you need to remove ReceiptParser and replace it with RevenueCat.

Reply:
One bummer with this is that SPM from Xcode might prompt you to install both by default, right?

Reply:
> I think we should just have these two be independent packages, which just happen to have the same files.

That's a great idea, which will simplify this tremendously.
I wanted to have a call with you to discuss the approach, but thanks for providing the best path forward already :D

Reply:
happy to chat about it whenever!